2048/2048/src/main/java/com/appcoins/eskills2048/LaunchActivity.java

```diff
@@ -92,6 +96,7 @@ private void resumeGame(LocalGameStatus localGameStatus) {
   }
 
   private void showCreateTicket(MatchDetails.Environment environment) {
+    this.match_environment = environment;
```

Comment:
usually `this` is only used when you want to distinguish class variables from input method arguments. in this case, you could just simply `matchEnvironment = environment;`

Reply:
fixed